GherkinSyncTool/GherkinSyncTool/NLog.config

```diff
@@ -6,6 +6,6 @@
     <target name="console" xsi:type="ColoredConsole" layout="${message} ${exception:format=Message,StackTrace}" />
   </targets>
   <rules>
-    <logger name="*" minlevel="Info" writeTo="console" />
+    <logger name="*" minlevel="Debug" writeTo="console" />
```

Comment:
Let's leave the Info level?

Reply:
I suggest we leave it Debug while we working on the tool - it's not very convenient to change it to Debug and back to info all the time

Reply:
It is possible to keep this file in the condition you like using git stash or Rider functionality. For the end-user, it should be info.